File: lib_base/src/main/java/com/trial/base/utils/LogUtils.java
package com.trial.base.utils;


import com.elvishew.xlog.XLog;

/**
 * <pre>
 *     @author : Trial
 *     @time   : 2021/07/09
 *     @desc   :
 *     @version: 1.0
 * </pre>
 */
public class LogUtils {

    public static void v(String msg) {
        XLog.v(msg);
    }

    public static void v(String tag, String msg) {
        XLog.v(tag, msg);
    }

    public static void i(String msg) {
        XLog.i(msg);
    }

    public static void i(String tag, String msg) {
        XLog.i(tag, msg);
    }

    public static void d(String msg) {
        XLog.d(msg);
    }

    public static void d(String tag, String msg) {
        XLog.d(tag, msg);
    }

    public static void e(String msg) {
        XLog.e(msg);
    }

    public static void e(String tag, String msg) {
        XLog.e(tag, msg);
    }

    public static void json(String msg) {
        XLog.json(msg);
    }

}

File: lib_base/src/main/java/com/trial/base/utils/LogUtils.kt
package com.trial.base.utils

import com.elvishew.xlog.XLog

/**
 * <pre>
 * @author : Trial
 * @time   : 11/25/21
 * @desc   :
 * @version: 1.0
</pre> *
 */
object LogUtils {
    fun v(msg: String?) {
        XLog.v(msg)
    }

    fun v(tag: String?, msg: String?) {
        XLog.tag(tag).v(msg)
    }

    fun i(msg: String?) {
        XLog.i(msg)
    }

    fun i(tag: String?, msg: String?) {
        XLog.tag(tag).i(msg)
    }

    fun d(msg: String?) {
        XLog.d(msg)
    }

    fun d(tag: String?, msg: String?) {
        XLog.tag(tag).d(msg)
    }

    fun e(msg: String?) {
        XLog.e(msg)
    }

    fun e(tag: String?, msg: String?) {
        XLog.tag(tag).e(msg)
    }

    fun json(msg: String?) {
        XLog.json(msg)
    }
}